### PR TITLE
chore(main): release 0.113.0

### PR DESCRIPTION
**Pull request by ooples**
:robot: I have created a release *beep* *boop*
---

### Features

* **training:** compiled backward + createGraph, DP-SGD, multi-slot persistent inputs ([#763](https://github.com/ooples/AiDotNet.Tensors/issues/763)) ([b7cf72e](https://github.com/ooples/AiDotNet.Tensors/commit/b7cf72e1838455f9d57b46e2e0d2e4f0c953e3d1))

### Performance

* **compile:** SIMD-vectorize global-L2 gradient clip in fused training ([#766](https://github.com/ooples/AiDotNet.Tensors/issues/766)) ([ea98267](https://github.com/ooples/AiDotNet.Tensors/commit/ea98267f85d5f762f07313563064627c952f3b5e))
* fused-training optimizer + arena suite (clip SIMD, moment pooling, AdamW fusion, bias-corr hoist, multi-tensor foreach) ([#770](https://github.com/ooples/AiDotNet.Tensors/issues/770)) ([7c8455d](https://github.com/ooples/AiDotNet.Tensors/commit/7c8455db631c4acc9abd9a6f4420b1e0251f400d))

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

## Summary by CodeRabbit

* **New Features**
  * Added support for training and multi-slot persistent inputs.
  * Added compiled backward and graph creation capabilities.
  * Added support for DP-SGD.

* **Performance**
  * Improved fused-training gradient clipping with SIMD vectorization.
  * Added fused-training optimizer and arena performance improvements.

* **Documentation**
  * Updated the changelog for version 0.113.0.

### CODE REVIEW

**Comment by ooples** [comment]
🤖 Created releases:

- [v0.113.0](https://github.com/ooples/AiDotNet.Tensors/releases/tag/v0.113.0)

:sunflower:
